### PR TITLE
[docs] add GitLab importer docs

### PR DESCRIPTION
**Pull request by tigrato**
This PR adds the docs required to setup an Access Graph GitLab integration.

Closes https://github.com/gravitational/access-graph/issues/962

### CODE REVIEW

File: docs/pages/access-controls/access-graph/gitlab.mdx [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+
+Teleport Policy’s Access Graph feature delves into the resources imported and their relationships, crafting a
+graphical representation thereof.
+
+
+## Prerequisites
+
+- A running Teleport Enterprise cluster v14.3.20/v15.3.1/v16.0.0 or later.
+- For self-hosted clusters, an updated `license.pem` with Teleport Policy enabled.
+- For self-hosted clusters, a running Access Graph node v1.20.0 or later.
```
Comment (jakule) [inline docs/pages/access-controls/access-graph/gitlab.mdx:53]: I'd suggest the latest as then we added a few UI fixes to the integration.
```suggestion
- For self-hosted clusters, a running Access Graph node v1.21.4 or later.
```

File: docs/pages/access-controls/access-graph/gitlab.mdx [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+Once all the necessary resources are fetched, Teleport pushes them to the
+Access Graph, ensuring that it remains updated with the latest information from your GitLab instance.
+
+### Importing resources
+
+Teleport Policy’s Access Graph feature delves into the resources imported and their relationships, crafting a
+graphical representation thereof.
+
+
+## Prerequisites
```
Comment (jakule) [inline docs/pages/access-controls/access-graph/gitlab.mdx:49]: Do we know what is the lowest supported GitLab self-hosted version?

Comment (tigrato) [reply docs/pages/access-controls/access-graph/gitlab.mdx:49]: Gitlab v4 was introduced in Gitlab version 6 and v3 was removed in Gitlab 9.

File: docs/pages/access-controls/access-graph/gitlab.mdx [hunk 3]
```diff
@@ -0,0 +1,119 @@
+---
+title: Discover GitLab Access Patterns with Teleport Policy
+description: Describes how to syncronize GitLab access patterns using Teleport Policy and Access Graph.
+---
+Teleport Policy streamlines and centralizes access management across your entire infrastructure, allowing you to view
+access relationships within seconds. You can see unified, up-to-date relationships and policies between all users, groups,
+and computing resources.
```
Comment (r0mant) [inline docs/pages/access-controls/access-graph/gitlab.mdx:7]: Nitpick but I would remove this paragraph. The less "happy talk" the better. The next one reads like a very good intro already.
```suggestion
```

File: docs/pages/access-controls/access-graph/gitlab.mdx [hunk 4]
```diff
@@ -0,0 +1,119 @@
...
+
+With Teleport Policy's Access Graph, you gain insights into access patterns within your GitLab account. By scanning all
+permissions, users, groups, and projects, it provides a visual representation to help enhance the permission model within
+your GitLab environment. This functionality enables you to answer queries such as:
+
+- What projects are accessible to users?
+- Which users have write permissions to projects?
+
+Access Graph is a feature of the [Teleport Policy](https://goteleport.com/platform/policy/) product
+available to Teleport Enterprise customers.
```
Comment (r0mant) [inline docs/pages/access-controls/access-graph/gitlab.mdx:17]:
```suggestion
available to Teleport Enterprise and Teleport Cloud customers.
```

File: docs/pages/access-controls/access-graph/gitlab.mdx [hunk 5]
```diff
@@ -0,0 +1,119 @@
...
+
+## Prerequisites
+
+- A running Teleport Enterprise cluster v14.3.20/v15.3.1/v16.0.0 or later.
+- For self-hosted clusters, an updated `license.pem` with Teleport Policy enabled.
+- For self-hosted clusters, a running Access Graph node v1.21.4 or later.
+Check [Access Graph page](../teleport-policy/getting-started-policy.mdx) for details on
+how to set up Access Graph.
+- The node running the Access Graph service must be reachable
+from Teleport Auth Service.
```
Comment (r0mant) [inline docs/pages/access-controls/access-graph/gitlab.mdx:57]: We should make it clear somehow that it's only for self-hosted and if you're using Cloud you don't have to do anything other than purchase it (unless it's a trial).

File: docs/pages/access-controls/access-graph/gitlab.mdx [hunk 6]
```diff
@@ -0,0 +1,119 @@
...
+Once the token is successfully validated, you'll be able to see the resources imported in Access Graph.
+
+## Step 3/3. View GitLab resources in Access Graph
+
+After the GitLab resources are imported, you can view them in the Access Graph page.
+The graph representation will show the relationships between users, groups, and projects within your GitLab instance.
+
+Users can have permissions to access a Group or Project. When a user has access to a Group, they inherit permissions
+to all projects and subprojects within that group.
+
```
Comment (r0mant) [inline docs/pages/access-controls/access-graph/gitlab.mdx:104]: Can we give a few examples of queries users can run to analyze Gitlab access paths?

**Review by r0mant — COMMENTED** [review]
Overall looks good, with a few suggestions.

**Review by jakule — APPROVED** [review]

File: docs/pages/access-controls/access-graph/gitlab.mdx (hunk 4 above)
Comment (tigrato) [reply docs/pages/access-controls/access-graph/gitlab.mdx]: Our docs linter denies using `Teleport Enterprise and Teleport Cloud`, instead it suggests `Teleport Enterprise edition`

**Review by r0mant — APPROVED** [review]

**Pull request merged**